filter.md

```diff
@@ -0,0 +1,217 @@
+# Filtering the output
+
+Tiger generates a lot of reports. This guide explains how to use filters to control what reports are printed.
+
+In the `ck3-tiger.conf` file, add a `filter` block. Inside the `filter`, add a `trigger` block.
+
+```
+# You can have at most 1 filter block.
+filter = {
+    # Whether to report about problems in vanilla game files.
+    # Setting this to 'yes' results in a LOT of spam.
+    # Optional boolean value, defaults to no.
+    show_vanilla = no
+
+    # Whether to report about problems in mods loaded via the load_mod sections.
+    # Optional boolean value, defaults to no.
+    show_loaded_mods = no
+    
+    # You can have at most 1 trigger block.
+    trigger = {
+        # The root trigger block is an AND-gate, just like in paradox script.
```

Comment:
Calling it the root trigger block is confusing for modders because "root" means something else. Maybe just say "You can have at most 1 trigger block. It is an implicit AND trigger, just like in paradox script."

Reply:
I'll see if I can rewrite the guide for normies. :)